packages/eas-cli/src/commands/account/login.ts

```diff
@@ -1,17 +1,32 @@
+import { Flags } from '@oclif/core';
+
 import EasCommand from '../../commandUtils/EasCommand';
 import Log from '../../log';
 
 export default class AccountLogin extends EasCommand {
   static override description = 'log in with your Expo account';
   static override aliases = ['login'];
 
+  static override flags = {
+    // can pass either --sso or -s
+    sso: Flags.boolean({
+      description: 'Login with SSO',
+      char: 's',
+      default: false,
+    }),
```

Comment:
```suggestion
    sso: Flags.boolean({
      description: 'Login with SSO',
      char: 's',
      default: false,
      hidden: true
    }),
```

Do the above to make it hidden

Reply:
Done, thanks